CONTRIBUTING.md

```diff
@@ -15,10 +14,17 @@ Thank you for your interest in contributing to the Uniswap interface! 🦄
 1. `yarn install`
 1. `yarn build`
 
+## Running tests
+
+1. Run unit tests: `yarn test`
+1. Run unit tests and show test coverage info for all tests: `yarn test --coverage --watchAll`
+1. Run unit tests and show test coverage info only for tests affected by your changes: `yarn test --coverage --watch`
+1. Run e2e cypress tests with visual interface: `yarn cypress open`
```

Comment:
In order to run cypress tests, you'll need to a build to test them off of.
You'll need to `yarn build:e2e` in order to get a build with infura keys that will work on your localhost (vs `yarn build`, which is for production), and then `yarn test:e2e` will open cypress (it aliases `cypress open`) with a server backing the tests. CI uses `yarn test:e2e:ci` to run without the cypress UI.

Reply:
Thanks! I added these instructions for running integration tests